lib/triggers/steamrepOnJoinTrigger.js

```diff
@@ -31,30 +31,31 @@ SteamrepOnJoinTrigger.prototype._respond = function(roomId,userToCheck) {
 	if (steamid) {
 		var that = this;
 		var steamrep={};
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['steamrep' is defined but never used.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969820463/issues/source?bid=1980036&fileBranchId=2848073#l33)